README.md

```diff
@@ -24,64 +24,355 @@
 
 # Endpoints
 
+## Emotions
 ## Get all emotions for a post
-get 'api/v1/post_emotions'
+get 'api/v1/emotions'
+
+ ``` 
+ {
+    "data": [
+        {
+            "id": "1",
+            "type": "emotion",
+            "attributes": {
+                "emotion": "Affectionate",
+                "definition": "(of a person) Having affection or warm regard; loving; fond."
+            }
+        },
+        {
+            "id": "2",
+            "type": "emotion",
+            "attributes": {
+                "emotion": "Confident",
+                "definition": "A person in whom one can confide or share one's secrets: a friend."
+            }
+        },
+        {
+            "id": "3",
+            "type": "emotion",
+            "attributes": {
+                "emotion": "Grateful",
+                "definition": "Appreciative; thankful."
+            }
+        }
+      ]
+  }
+  ``` 
+
+
+## Users
 
-<img src="./app/assets/images/emotions_endpoint.jpg" alt="This is what we expect to return when we query for a list of emotions for the post page" width="600" height="350" />
-
-
-## Get all friends of a user with a status of pending
-get 'api/v1/friends?request_status=pending'
+## Get a specific user based on their google id
+get 'api/v1/users?search=<google_id>'
+```
+{
+    "data": {
+        "id": "6",
+        "type": "user",
+        "attributes": {
+            "name": "Hisoka Morow",
+            "email": "hisoka.morow@meteorcity.gov",
+            "google_id": "91239464"
+        }
+    }
+}
+```
+## returns this if no user is found with that google id
+
+```
+{
+    "data": []
+}
+```
+
+## Search for a user using their email address
+get '/api/v1/users?by_email=#{email}'
+or
+get '/api/v2/users?email=<email>'
+
+
+```{
+    "data": {
+        "id": "5",
+        "type": "user",
+        "attributes": {
+            "name": "Gon Freecss"
+        }
+    }
+}
+```
+## returns this if no user is found with that email address
+
+```
+{
+    "data": []
+}
+```
+
+## returns a user when searching with both name and google_id
+get '/api/v2/users?name=<name>&email=<email>&google_id=<google_id>'
```

Comment:
I think this one is a POST on v2 for creating a new user I don't think we have a search GET path that works with all these params we only do a search GET by email. I'm double-checking myself on this though.

Reply:
Just double checked code base, and we only have this as a POST

Reply:
Moved this down to the Create a user section for v1 or v2 options